utils/load_utils.py

```diff
@@ -194,13 +204,13 @@ def load_depmap(
 
 
 def load_prism(
-    top_dir,
-    data_dir="prism/data",
-    secondary_screen=False,
-    load_cell_info=False,
-    load_treatment_info=False,
-    only_get_cells=False,
-):
+    top_dir: str,
+    data_dir: str = "prism/data",
+    secondary_screen: bool = False,
+    load_cell_info: bool = False,
+    load_treatment_info: bool = False,
+    only_get_cells: bool = False,
+) -> List[pd.DataFrame]:
```

Comment:
Based on my understanding of the potential returns, consider using a `Union` here to specify multiple return types (this also requires an additional `typing` import). The same may apply for `load_nci60`.

Note: this syntax is going through changes as of Python 3.10, so please use what makes most sense here ([reference](https://docs.python.org/3/library/typing.html#typing.Union)). Noticing the environment files I didn't see a suggested Python version, so this may be something to consider specifying as well (within the environment files or somewhere in the readme/docs).

```suggestion
) -> Union[List[pd.DataFrame], pd.DataFrame]:
```

Reply:
nice!